create-parser.sh

```diff
@@ -0,0 +1,41 @@
+#/bin/sh
+
+name=$1;
+
+if [ "$name" = "" ]
+then
+  echo "Please name your parser (name needs to be in kebab-case)";
+  exit;
+elif [ -d "./parsers/$name" ]
+then
+  echo "The parser $name already exist. Please choose another name.";
```

Comment:
Replace "The parser $name already exist. Please choose another name." by "A parser with the same name already exists. Please choose another name."